### PR TITLE
Screenshot

### PR DESCRIPTION
**Pull request by negar-abbasi**
- Add a download button for a screenshot of the visual transactions.

Screenshot 2024-08-23 at 9 48 50 AM

### CODE REVIEW

File: src/features/transactions-graph/components/transactions-graph.tsx [hunk 1]
```diff
@@ -15,63 +19,89 @@ export function TransactionsGraph({ transactionsGraphData }: Props) {
...
+            <Horizontal key={index} verticals={verticals} horizontal={row} />
+          ))}
         </div>
-        {horizontals.map((row, index) => (
-          <Horizontal key={index} verticals={verticals} horizontal={row} />
-        ))}
       </div>
-    </div>
+      <div className="sticky bottom-0 left-full z-50 flex size-0 overflow-visible">
+        <Button id="download-transactions-visual" className="absolute bottom-1 right-0 w-32" variant="outline" onClick={downloadImage}>
```
Comment (PatrickDinh) [inline src/features/transactions-graph/components/transactions-graph.tsx:100]: out of curiosity, do you need `id="download-transactions-visual"` here? It doesn't seem to be referenced anywhere else.

File: src/features/transactions-graph/components/transactions-graph.tsx [hunk 2]
```diff
@@ -15,63 +19,89 @@ export function TransactionsGraph({ transactionsGraphData }: Props) {
...
+    const link = document.createElement('a')
+    link.href = dataUrl
+    link.setAttribute('download', 'transactions-visual.png')
+    // TODO: This approach won't work in Tauri, so we'll need to handle with Tauri's APIs
+    link.click()
+  }, [])
+
   return (
-    // Don't change this id value, it's used by a bot Alessandro is building.
-    <div id="visual-transactions">
```
Comment (PatrickDinh) [inline src/features/transactions-graph/components/transactions-graph.tsx:20]: have you confirmed that the id isn't needed anymore?

Comment (negar-abbasi) [reply src/features/transactions-graph/components/transactions-graph.tsx:20]: yes, only the id on the button will be used.

File: src/features/transactions-graph/components/transactions-graph.tsx (hunk 1 above)
Comment (negar-abbasi) [reply src/features/transactions-graph/components/transactions-graph.tsx:100]: Added a comment for clarity.

**Review by PatrickDinh — APPROVED** [review]

**Review by neilcampbell — APPROVED** [review]

**Pull request merged**